src/main/java/org/jenkinsci/plugins/codescene/DeltaAnalysis.java

```diff
@@ -24,16 +24,13 @@ public DeltaAnalysis(Configuration config) {
         this.config = config;
     }
 
-    public DeltaAnalysisResult runOn(final Commits commits) throws RemoteAnalysisException {
+    public DeltaAnalysisResult runOn(final Commits commits) throws RemoteAnalysisException, IOException {
         final DeltaAnalysisRequest payload = new DeltaAnalysisRequest(commits, config);
 
         try {
             return synchronousRequestWith(payload, commits);
         } catch (URISyntaxException e) {
             throw new IllegalArgumentException("The configured CodeScene URL isn't valid", e);
-        } catch (IOException e) {
-            throw new RemoteAnalysisException("Failed to send request to CodeScene at " + config.codeSceneUrl().toString()
-                    + "\n  cause: " + e.getMessage(), e);
```

Comment:
It would be useful to capture why IOException is not captured anymore and instead thrown here - perhaps in a comment or a javadoc for this method. 
I will most likely forget soon why this has been changed :).

Reply:
java doc added